src/toil/batchSystems/mesos/__init__.py

```diff
@@ -44,4 +44,8 @@
     # A dictionary with additional environment variables to be set on the worker process
     'environment',
     # A named tuple containing all the required info for cleaning up the worker node
-    'workerCleanupInfo'))
+    'workerCleanupInfo',
+    # A string containing the resolved mesos master address
```

Comment:
And "resolved" indicates that it is always an IP address, never a host name. Is that really true?